frontend/src/assets/translations/nl.json

```diff
@@ -0,0 +1,30 @@
+{
+  "about": "Over ons",
+  "volunteers": "Vrijwilligers",
+  "contactus": "Contacteer Ons",
+  "signup": "Aanmelden",
+  "login": "Inloggen",
+  "findyourvolunteer": "VIND JE BENEDENDE",
+  "volunteerswhocanhelp": "VRIJWILLIGERS WELKE HET KAN HELPEN",
+  "skill": "Vaardigheid",
+  "city": "Stad",
+  "language": "Taal",
+  "whoarewe": "WIE ZIJN WE?",
+  "aboutbehelp1": "BeHelp is een dienst die reisden en asielverzoekers verbindt met vrijwilligers die zich kunnen delen met hun ervaring en vaardigheden pro bono.",
```

Comment:
behelp is een dienst dat verbindt azielzoekers en vluchtelingen met lokale vrijwilligers die hun kennis en vaardigheden willen delen.
??????

Reply:
?

Reply:
its means  that im not sure that is really beautifull thranslation)))) but normally is ok

Reply:
i ll do little step by step ok? according when my internet have connection)